versions.tf

```diff
@@ -6,13 +6,5 @@ terraform {
       source  = "hashicorp/aws"
       version = ">= 3.0"
```

Comment:
I think this needs to be https://github.com/hashicorp/terraform-provider-aws/releases/tag/v3.64.0 at a minimum

Reply:
wouldn't this > sign pull the latest version?

Reply:
Yes, but the question is whether or not we want to explicit about it or implicit about it.

Reply:
If we aren't explicit about it anywhere else, then that's fine. Just wanted to point this out.

Reply:
updated the pr description